addOns/grpc/CHANGELOG.md

```diff
@@ -5,5 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
-- First version.
-
+## 0.1.0 - 2024-05-17
```

Comment:
You should leave this, it'll be done by the release workflow.

Reply:
I mean leave the unreleased heading, remove this one, leave the added and notes

Reply:
The "First version" entry can be removed too, that's implicit by being the first release.

Reply:
The `## Unreleased` should be kept.